control

```diff
@@ -8,9 +8,9 @@ Description: Adds custom devices to VeriStand.
 XB-Eula: eula-ni-standard,
 Priority: standard
 Homepage: http://www.ni.com
-Recommends: ni-scan-engine-veristand-2019-support (>=20.0.0), ni-scan-engine-veristand-2020-support (>=20.0.0), ni-synchronization-veristand-2019-support (>=20.0.0), ni-synchronization-veristand-2020-support (>=20.0.0), ni-engine-simulation-veristand-2019-support (>=20.0.0), ni-engine-simulation-veristand-2020-support (>=20.0.0), ni-slsc12201-veristand-2019-support (>=20.0.0), ni-slsc12201-veristand-2020-support (>=20.0.0), ni-slsc-eds-veristand-2019-support (>=20.0.0), ni-slsc-eds-veristand-2020-support (>=20.0.0), ni-routing-and-faulting-veristand-2019-support (>=20.0.0), ni-routing-and-faulting-veristand-2020-support (>=20.0.0), ni-slsc-switch-veristand-2019-support (>=20.0.0), ni-slsc-switch-veristand-2020-support (>=20.0.0), ni-embedded-data-logger-veristand-2020-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2019-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2020-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-examples (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-examples (>= 20.0.0), ni-instrument-addon-veristand-2019-support (>=20.0.0), ni-instrument-addon-veristand-2020-support (>=20.0.0),
+Recommends: ni-scan-engine-veristand-2019-support (>=20.0.0), ni-scan-engine-veristand-2020-support (>=20.0.0), ni-synchronization-veristand-2019-support (>=20.0.0), ni-synchronization-veristand-2020-support (>=20.0.0), ni-engine-simulation-veristand-2019-support (>=20.0.0), ni-engine-simulation-veristand-2020-support (>=20.0.0), ni-slsc12201-veristand-2019-support (>=20.0.0), ni-slsc12201-veristand-2020-support (>=20.0.0), ni-slsc-eds-veristand-2019-support (>=20.0.0), ni-slsc-eds-veristand-2020-support (>=20.0.0), ni-routing-and-faulting-veristand-2019-support (>=20.0.0), ni-routing-and-faulting-veristand-2020-support (>=20.0.0), ni-slsc-switch-veristand-2019-support (>=20.0.0), ni-slsc-switch-veristand-2020-support (>=20.0.0), ni-embedded-data-logger-veristand-2020-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2019-labview-support (>= 20.0.0), ni-slsc-switch-veristand-2020-labview-support (>= 20.0.0), ni-routing-and-faulting-veristand-2019-labview-examples (>= 20.0.0), ni-routing-and-faulting-veristand-2020-labview-examples (>= 20.0.0), ni-instrument-addon-veristand-2019-support (>=20.0.0), ni-instrument-addon-veristand-2020-support (>=20.0.0), ni-fpga-addon-2019-support (>=20.0.0), ni-fpga-addon-2020-support (>=20.0.0),
```

Comment:
The package name is ni-fpga-addon-veristand-2019-support

Reply:
Feedback was implemented!